license-maven-plugin-git/src/main/java/com/mycila/maven/plugin/license/git/GitLookup.java

```diff
@@ -69,9 +66,10 @@
    * @param dateSource        where to read the commit dates from - committer date or author date
    * @param timeZone          the time zone if {@code dateSource} is {@link DateSource#COMMITER}; otherwise must be {@code null}.
    * @param checkCommitsCount
+   * @param commitsToIgnore
```

Comment:
Probably should explain what is expected in this argument.

Reply:
Roger